.github/workflows/wheel.yml

```diff
@@ -61,13 +69,38 @@ jobs:
       - name: Install Python dependencies
         run: python -m pip install cibuildwheel twine
 
+      - name: Install boost if ubuntu
+        if: ${{ contains(matrix.os, 'ubuntu') }}
+        run: sudo apt update && sudo apt install libboost-dev
```

Comment:
If we do not run `apt upgrade`, we can remove `apt update`, right?

Reply:
Really? I think `libboost-dev` may not found in the repository without `apt update`, but actually it is always found on GitHub Actions Runner.